XenoHolographic-it/IAR-SDK-Sample-iOS
# IAR-4955 Get markers by Location [Core]

# Overview

- Add Location Market list;
- Allow to change the current coordination and update the list;
- Open a new view with details when a marker is selected;

# Resources

Jira: https://iversoft.atlassian.net/browse/IAR-4955

# Technical Considerations

- The input for the new Latitude and Longitude is using keyboard type `.numbersAndPunctuation`
- The input for the new Latitude and Longitude needs to follow the rule: `Latitude;Longitude` to work properly.

# Known Issues

none

# Testing Instructions

- Select menu Location Markers.
- To see its details, select one of the list.
- To search for those available in an area, click Button on top.
